src/xrpld/app/tx/detail/InvariantCheck.cpp

```diff
@@ -1120,4 +1123,79 @@ ValidMPTIssuance::finalize(
         mptokensCreated_ == 0 && mptokensDeleted_ == 0;
 }
 
+//------------------------------------------------------------------------------
+
+void
+ValidPermissionedDomain::visitEntry(
+    bool,
+    std::shared_ptr<SLE const> const& before,
+    std::shared_ptr<SLE const> const& after)
+{
+    if (after->getType() != ltPERMISSIONED_DOMAIN)
```

Comment:
Looks like the norm in invariant checks is to check the `before` _and_ `after`

Reply:
This doesn't look to have changed

Reply:
restored